features/in_onderzoek.feature

```diff
@@ -445,215 +531,246 @@ Functionaliteit: in onderzoek
-      | aanduidingGegevensInOnderzoek | 080000   |
-      | datumIngangOnderzoek          | 20220307 |
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                     | waarde   |
+      | straat                                   | spui     |
+      | huisnummer                               | 70       |
+      | aanduiding gegevens in onderzoek (83.10) | 080000   |
+      | datum ingang onderzoek (83.20)           | 20220307 |
       Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                             |
-      | type                | RaadpleegMetBurgerservicenummer    |
-      | burgerservicenummer | 555550001                          |
-      | fields              | burgerservicenummer,verblijfplaats |
-      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats.inOnderzoek' gegevens
-      | naam                             | waarde     |
-      | aanduidingBijHuisnummer          | true       |
-      | datumIngangGeldigheid            | true       |
-      | datumVestigingInNederland        | true       |
-      | functieAdres                     | true       |
-      | huisletter                       | true       |
-      | huisnummer                       | true       |
-      | huisnummertoevoeging             | true       |
-      | nummeraanduidingIdentificatie    | true       |
-      | adresseerbaarObjectIdentificatie | true       |
-      | landVanwaarIngeschreven          | true       |
-      | straat                           | true       |
-      | postcode                         | true       |
-      | korteNaam                        | true       |
-      | woonplaats                       | true       |
-      | datumIngangOnderzoek.datum       | 2022-03-07 |
-      | datumIngangOnderzoek.type        | Datum      |
-      | datumIngangOnderzoek.langFormaat | 7 maart 2022 |
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | verblijfplaats                  |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde       |
+      | type                                                       | Adres        |
+      | inOnderzoek.adresseerbaarObjectIdentificatie               | true         |
+      | inOnderzoek.datumIngangGeldigheid                          | true         |
+      | inOnderzoek.datumVan                                       | true         |
+      | inOnderzoek.functieAdres                                   | true         |
+      | inOnderzoek.nummeraanduidingIdentificatie                  | true         |
```

Comment:
Klopt het dat 'verblijfplaats.inOnderzoek' ook een 'type' veld moet hebben om het 'type' veld in 'verblijfplaats' op onderzoek te kunnen zetten?
Zo niet dan moet deze in de OAS specs nog verwijderd worden uit alle 4 de componenttypes.
Zo ja, dan neem ik aan dat dat veld ook hier in de response opgenomen moet worden.

Hetzelfde geldt voor de volgende 2 scenario's.

Reply:
klopt, in de tabel ontbreekt inOnderzoek.type

Reply:
ik kom dat niet tegen in het 'in onderzoek.xls' bestand en ook niet in de initiele scenario. Ik weet dus niet wanneer inOnderzoek.type op true moet worden gezet. Kunnen jullie dat specificeren?

Reply:
'in onderzoek.xls' moet nog aangepast worden aan de huidige status van de specificaties.

Reply:
inOnderzoek.type moet worden toegevoegd in de Dan stap

Reply:
toegevoegd